lib/index.js

```diff
@@ -0,0 +1,27 @@
+var _ = require('lodash');
+
+var permissions = [];
+
+function addPermissions(permission) {
+    if (permissions.indexOf(permission) === -1) {
```

Comment:
This is computationally expensive. Instead, would be best to do something like store permissions in an associative array with both key and value being the permissionName string. Then you generate the array by doing Object.keys on that permissions object. In this case, addPermissions would just overwrite the old permission string and exec in O(1) since you wouldn't have to check the input of your addPermissions call against all old permissions.